src/components/Aframe/my-loader.js

```diff
@@ -57,6 +54,12 @@ AFRAME.registerComponent("myloader", {
     channel: { type: "number", default: 6 },
     cameraState: { type: "string", default: "" },
     myMeshPosition: { type: "vec3", default: "" },
+    path: { type: "string", default: "" },
+    slices: { type: "number", default: 55 },
+    extension: { type: "string", default: "png" },
```

Comment:
is the extension needed? or is that included in the path?

Reply:
Removed it and nothing broke so I'll go with no 😄